src/MediaPlayer/MediaPlayer.cs

```diff
@@ -119,18 +117,25 @@ public enum MPMediaType : nuint_compat_int {
 		AnyAudio     = 0x00ff,
 		
 		[iOS (5,0)]
+		[Mac (10,12,2)]
```

Comment:
If all members have a identical availability attribute then it should be moved to the type to reduce the metadata